engine/src/cmd/collide2/Ice/IceTypes.h

```diff
@@ -45,38 +46,38 @@
 	#define null				0														//!< our own NULL pointer
 
 	// Custom types used in ICE
-	typedef signed char			sbyte;		//!< sizeof(sbyte)	must be 1
-	typedef unsigned char		ubyte;		//!< sizeof(ubyte)	must be 1
-	typedef signed short		sword;		//!< sizeof(sword)	must be 2
-	typedef unsigned short		uword;		//!< sizeof(uword)	must be 2
-	typedef signed int			sdword;		//!< sizeof(sdword)	must be 4
-	typedef unsigned int		udword;		//!< sizeof(udword)	must be 4
-	typedef int64		sqword;		//!< sizeof(sqword)	must be 8
-	typedef uint64	uqword;		//!< sizeof(uqword)	must be 8
-	typedef float				float32;	//!< sizeof(float32)	must be 4
-	typedef double				float64;	//!< sizeof(float64)	must be 4
-
-	// ICE_COMPILE_TIME_ASSERT(sizeof(bool)==1);	// ...otherwise things might fail with VC++ 4.2 !
-	ICE_COMPILE_TIME_ASSERT(sizeof(ubyte)==1);
-	ICE_COMPILE_TIME_ASSERT(sizeof(sbyte)==1);
-	ICE_COMPILE_TIME_ASSERT(sizeof(sword)==2);
-	ICE_COMPILE_TIME_ASSERT(sizeof(uword)==2);
-	ICE_COMPILE_TIME_ASSERT(sizeof(udword)==4);
-	ICE_COMPILE_TIME_ASSERT(sizeof(sdword)==4);
-	ICE_COMPILE_TIME_ASSERT(sizeof(uqword)==8);
-	ICE_COMPILE_TIME_ASSERT(sizeof(sqword)==8);
+    typedef int8_t              ice_sbyte;      //!< sizeof(ice_sbyte)      must be 1
+    typedef uint8_t             ice_ubyte;      //!< sizeof(ice_ubyte)      must be 1
+    typedef int16_t             ice_sword;      //!< sizeof(ice_sword)      must be 2
+    typedef uint16_t            ice_uword;      //!< sizeof(ice_uword)      must be 2
+    typedef int32_t             ice_sdword;     //!< sizeof(ice_sdword)     must be 4
+    typedef uint32_t            ice_udword;     //!< sizeof(ice_udword)     must be 4
+    typedef int64_t             ice_sqword;     //!< sizeof(ice_sqword)	    must be 8
+    typedef uint64_t            ice_uqword;     //!< sizeof(ice_uqword)     must be 8
+    typedef float               ice_float32;    //!< sizeof(ice_float32)    must be 4
+    typedef double              ice_float64;    //!< sizeof(ice_float64)    must be 4
+
+    // ICE_COMPILE_TIME_ASSERT(sizeof(bool)==1);	// ...otherwise things might fail with VC++ 4.2 !
+    ICE_COMPILE_TIME_ASSERT(sizeof(ice_ubyte)==1);
+    ICE_COMPILE_TIME_ASSERT(sizeof(ice_sbyte)==1);
+    ICE_COMPILE_TIME_ASSERT(sizeof(ice_sword)==2);
+    ICE_COMPILE_TIME_ASSERT(sizeof(ice_uword)==2);
+    ICE_COMPILE_TIME_ASSERT(sizeof(ice_udword)==4);
+    ICE_COMPILE_TIME_ASSERT(sizeof(ice_sdword)==4);
+    ICE_COMPILE_TIME_ASSERT(sizeof(ice_uqword)==8);
+    ICE_COMPILE_TIME_ASSERT(sizeof(ice_sqword)==8);
```

Comment:
Here's where I renamed these typedefs. Also, please, let's use the fixed-width integer types that C++11 provides, instead of making the faulty assumptions that the code made before.